pkg/datastore/migrate.go

```diff
@@ -15,89 +13,44 @@ import (
 )
 
 // MigrateDB applies necessary DB Migrations.
-func (ds *Datastore) MigrateDB(healthServer *health.Server) error {
+func MigrateDB(log *zap.SugaredLogger, db *sqlx.DB, healthServer *health.Server) error {
 
 	m, err := migrator.New(
 		migrator.Migrations(
 			// migrations will be applied and remembered in this order, so always add new migrations below if necessary
 			&migrator.Migration{
-				Name: "Consolidate History for Tenant and Project",
+				Name: "Sample Migration for Tenant",
```

Comment:
I think a cleaner way would be to keep at least the name of the old migration and remove the migration code including a comment if it's not worth to fix it. Then add a breaking change section to this PR that includes a hint on updating to latest minor release before rolling out this version.

Reply:
I think there is no point keep this backward compatible because we only have small amount of installations and this changes was already done 2 years ago.